### PR TITLE
Delete multiple confirmations

### PR DESCRIPTION
**Pull request by leventem**
### What does this PR do? How does it affect users?
- New version of the provider: MessageStore interface has deleteMultiConfIfNoMatchingMsg method for confirmation cleanup.
- other than that minor inner cleanup for parameters

@stratiator could you please check?

cc @privateblue

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,7 +2,7 @@ name := "amqp-client-provider"
 
 organization := "com.kinja"
 
-version := "0.4.0" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
+version := "0.5.0" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
```
Comment (privateblue) [inline build.sbt:5]: why is this a minor upgrade and not a patch?

Comment (leventem) [inline build.sbt:5]: It changes the MessageStore interface that we implement in an other repo (for now).

**Comment by andris-szatmari** [comment]
LGTM!

File: build.sbt (hunk 1 above)
Comment (andris-szatmari) [inline build.sbt:5]: Configuration requirements are changed. It will break existing clients if they are rebuilt, so it's not a minor but a major update.

**Pull request merged**